db/seeds/event_questions.rb

```diff
@@ -19,20 +19,4 @@
   disclosure: :optional,
   event_type: Event::Course.sti_name,
   multiple_choices: true
-})
-
-Event::Question.seed_global({
-  question: 'Den schub (Ordner mit fünf schub-Broschüren, digital unter jubla.ch/schub)...',
-  choices: 'habe ich bereits und nehme ihn mit in den Kurs, leihe ich von jmd aus und nehme ihn mit in den Kurs (je neuer die Auflage desto besser), habe ich nicht/leihe ich nicht aus: bestelle ich hiermit als Ordner (Kosten: CHF 27.-)',
-  disclosure: :required,
-  event_type: Event::Course.sti_name,
-  multiple_choices: false
-})
-
-Event::Question.seed_global({
-  question: 'Das meisterwerk (Handbuch der Mindestkenntnisse Jubla-Technik, digital unter jubla.ch/jublatechnik)...',
-  choices: 'besitze ich bereits und nehme es mit in den Kurs (Auflage ab 2017), leihe ich von jemandem aus der Schar aus und nehme es mit in den Kurs (Auflage ab 2017), habe ich nicht und hätte es gerne als eigenes analoges Handbuch (ohne Kosten)',
-  disclosure: :required,
-  event_type: Event::Course.sti_name,
-  multiple_choices: false
```

Comment:
Damit werden die Frage-Vorlagen in Zukunft nicht mehr neu erstellt, falls sie fehlen. Aber aus der Datenbank verschwinden die Vorlagen damit noch nicht. Es braucht noch eine Datenmigration dafür, siehe Tech Spec in #221.